server/routes/api.js

```diff
@@ -2,9 +2,16 @@ const express = require('express');
 
 const router = express.Router();
 
-/* GET users listing. */
-router.get('/', (req, res) => {
-  res.json({ page: 'Api' });
-});
+module.exports = (services) => {
+  /* GET users listing. */
```

Comment:
This doesn't look like user listing anymore, so please fix or remove this comment.

Reply:
@bill-feng Thanks! I fixed the comment!:)